src/Components/UniversalChatbot/UniversalHeader.scss

```diff
@@ -1,4 +1,11 @@
 .arh__header {
   padding-top: var(--pf-t--global--spacer--md);
   padding-bottom: var(--pf-t--global--spacer--md);
+  & .pf-chatbot__menu .pf-chatbot__button--toggle-menu{
+    align-items: center;
+  }
+  &.pf-chatbot__header.pf-v6-u-p-sm .pf-chatbot__title img {
+    max-height: 32px;
+    margin-top: 2px;
```

Comment:
We currently have a max-height set to 24px on the img in PF: https://github.com/patternfly/chatbot/blob/main/packages/module/src/ChatbotHeader/ChatbotHeader.scss#L151

This was too small and made the logo very hard to see.

The `margin-top: 2px` is required as our SVG viewbox also contains a shadow, however this makes the icon appear shifted upwards compared to everything else. Once we update the SVG we can remove this.